themes/bubble-wrap-icons.yaml

```diff
@@ -1,12 +1,30 @@
+global:
+    # To allow standalone icon use let's define core terms that
+    # are otherwise defined in the root {style}-style.yaml,
+    # themes/color-{theme}.yaml or themes/label-{step} files.
+    # This also means {style}-icons.yaml should be imported first.
+    icon_size:          [[13, '14px'], [16, '18px'], [18, '19px']]
+    text_font_family:   'Quicksand'
+    #black_color:        [0.00,0.00,0.00]
+    #dark_color:         [0.250,0.250,0.250]
+    #white_color:        [1.00,1.00,1.00]
+    text_stroke:        [0.870,0.870,0.870]
+
+    # To make Refill's colorize-able icons play nice with other styles.
+    # Bubble Wrap defaults to "" null (don't color). If colorized icons are desired
+    # than set sdk_icon_color_style property value to "colorized_icons",
+    # or import the Refill color theme *after* Bubble Wrap icons
+    sdk_icon_color_style:
+
 styles:
     mapzen_icon_library:
         base: points
         texture: mapzen_icon_library
         # To make Refill's colorize-able icons play nice with other styles
-        mix:             # intentionally blank, is parsed as null (no mix)
+        mix: global.sdk_icon_color_style
```

Comment:
Hmm one does not need to specify this. The style property merging process will make sure to get the `mix` property from an import.

Reply:
The same will apply to all other base style (other PRs).

Reply:
Got it. Fixed in https://github.com/tangrams/bubble-wrap/pull/258/commits/36315b87f9f1d6e559aedbe31ced91fbc653810d and all the other PRs have also been updated.

All the example imports in the blog post still work!

Reply:
Looks good to me.